packages/amplify-e2e-core/src/categories/lambda-function.ts

```diff
@@ -206,6 +208,19 @@ export const addLambdaTrigger = (chain: ExecutionContext, cwd: string, settings:
   }
 };
 
+export const unCommentCorsHeader = async (cwd: string, functionName: string): Promise<void> => {
+  const codeHandlerPath = path.join(cwd, 'amplify', 'backend', 'function', functionName, 'src', 'index.js');
+  const code = fs.readFileSync(codeHandlerPath, { encoding: 'utf8' });
+  const uncommentedCode = code
+    .replace('//  headers: {', '    headers: {')
+    .replace('//      "Access-Control-Allow-Origin": "*"', '        "Access-Control-Allow-Origin": "*"')
+    .replace('//  },', '    },');
+  fs.writeFileSync(codeHandlerPath, uncommentedCode, {
+    flag: 'w',
+    encoding: 'utf8',
+  });
+};
+
```

Comment:
Why don't we add an helper function that determines if we want to allow CORs or not based on a global/env variable. And in the code turn set that global variable instead of using string replace which can be error prone

Reply:
Done